src/browser/base/content/zen-styles/zen-tabs/vertical-tabs.css

```diff
@@ -714,4 +711,9 @@
       }
     }
   }
+  @media (-moz-bool-pref: 'zen.view.compact') {
```

Comment:
Please move this to `zen-compact-mode.css`

Reply:
Done.